src/main/services/opds.ts

```diff
@@ -96,6 +98,7 @@ export class OpdsService {
 
         return httpGet(url, {
             timeout: 10000,
+            rejectUnauthorized: (_NODE_ENV === "development") ? false : true,
```

Comment:
Good idea, although I would suggest adding this in the `httpGet` implementation as a default in `{}`:
https://github.com/readium/readium-desktop/blob/17c866a684fe3ac2ef6dcd46723856fb8315507e/src/common/utils/http.ts#L61-L70

...and also in the download service:

https://github.com/readium/readium-desktop/blob/17c866a684fe3ac2ef6dcd46723856fb8315507e/src/main/services/downloader.ts#L136-L145

This way, this could cover a complete OPDS/LCP/LSD HTTP test environment using self-signed certs.

Reply:
Okay, can do.  Didn't know how comprehensive it should be, given that `http.ts` is in the util namespace, if that should be expected to be environment-aware.